lib/bsd/pty.c

```diff
@@ -67,7 +67,7 @@ static void
 mkserver(void)
 {
 	int fd, i;
-	char * const argv[3]={"ptyfs", 0};
+	const char *argv[3]={"ptyfs", 0};
```

Comment:
Linux have exec family divided betwwen both kind of args. I think it doesn't hurt if exec syscall is always right. For now seems to work. I changed to char * const* because I remember hundred of warnings during gcc building. My commit is in history so if I must back to it, I'll do.